addons/website_slides/wizard/slide_channel_invite_views.xml

```diff
@@ -6,34 +6,47 @@
+                        <group col="1">
+                            <group col="2" attrs="{'invisible':['|', ('with_enrollment', '=', True), ('channel_visibility', '!=', 'public')]}">
+                                <field name="channel_invite_url" readonly="1" widget="CopyClipboardChar"/>
+                                <field name="send_email" widget="boolean_toggle"/>
+                            </group>
+                            <group col="2" attrs="{'invisible':[('send_email', '=', False)]}">
+                                <field name="partner_ids"
+                                    widget="many2many_tags_email"
+                                    placeholder="Add contacts..."
+                                    attrs="{'required':[('send_email', '=', True)]}"
+                                    context="{'force_email':True, 'show_email':True, 'no_create_edit': True, 'no_quick_create': True}"/>
+                            </group>
+                            <group col="2" attrs="{'invisible':[('send_email', '=', False)]}">
+                                <field name="lang" invisible="1"/>
+                                <field name="render_model" invisible="1"/>
+                                <field name="subject" placeholder="Subject..."/>
                             </group>
-                            <group>
-                                <field name="template_id" label="Use template" context="{'default_model': 'slide.channel.partner'}"/>
+                            <field name="can_edit_body" invisible="1"/>
+                            <field name="body" class="oe-bordered-editor" options="{'style-inline': true}" attrs="{'readonly': [('can_edit_body', '=', False)], 'invisible':[('send_email', '=', False)]}" force_save="1"/>
+                            <group attrs="{'invisible':[('send_email', '=', False)]}">
+                                <group>
+                                    <field name="attachment_ids" widget="many2many_binary"/>
+                                </group>
+                                <group>
+                                    <field name="template_id" label="Use template" context="{'default_model': 'slide.channel.partner'}"/>
+                                </group>
                             </group>
                         </group>
-                    </group>
+                    </sheet>
                     <footer>
-                        <button string="Send" name="action_invite" type="object" class="btn-primary" data-hotkey="q"/>
-                        <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z"/>
+                        <button string="Send" attrs="{'invisible':[('send_email', '=', False)]}" name="action_invite" type="object" class="btn-primary" data-hotkey="q"/>
+                        <button string="Close" attrs="{'invisible':[('send_email', '=', True)]}" class="btn-secondary" special="cancel" data-hotkey="z"/>
```

Comment:
What is the wizard doing when send_email is False, as there is no action except cancel ?